modules/selection/main/index.coffee

```diff
@@ -16,9 +16,41 @@ getMethod = (node, methodName) ->
   else
     Element.prototype[methodName]
 
+getMatches = (node) ->
+  # XXX: use hx.vendor
+  matchesMethod =
+    if node.matches then 'matches'
+    else if node.matchesSelector then 'matchesSelector'
+    else if node.mozMatchesSelector then 'mozMatchesSelector'
+    else if node.webkitMatchesSelector then 'webkitMatchesSelector'
+    else if node.oMatchesSelector then 'oMatchesSelector'
+    else if node.msMatchesSelector then 'msMatchesSelector'
```

Comment:
Potentially, although it looks like it would have to still check for 'node.matches' first and then use hx.vendor if that doesn't exist